js/search.js

```diff
@@ -342,7 +341,7 @@
 		buf += '</span> ';
 
 		// abilities
-		if (gen >= 3) {
+		if (gen >= 3 && this.engine.formatType !== 'letsgo') {
 			var abilities = Dex.forGen(gen).species.get(id).abilities;
```

Comment:
```suggestion
			var abilities = this.engine.dex.species.get(id).abilities;
```

Reply:
also would you mind changing `(abilities['1'] ? abilities['1'] : '')` to `(abilities['1'] || '')` (line 368 of search.js)? its been bothering me for a bit lol.

Reply:
i had these changes in another pr im making soon but w/e